marimo/_plugins/ui/_impl/tables/table_manager.py

```diff
@@ -26,6 +26,8 @@
 class TableManager(abc.ABC, Generic[T]):
     DEFAULT_ROW_LIMIT = 20_000
     DEFAULT_COL_LIMIT = 100
+    DEFAULT_SUMMARY_ROW_LIMIT = 1_000_000
```

Comment:
could we add a comment here or link to the ticket, just so no one changes this without investigating.

Reply:
Comments are added. I also added a comment for `DEFAULT_ROW_LIMIT` and `DEFAULT_COL_LIMIT`, and please take a look. Thanks.